docs/dataset/object-detection.md

```diff
@@ -0,0 +1,366 @@
+    ""label"": ""car"", ""data_type"": ""ANNOTATION/BOUNDING_BOX""},
+    {""top_left"": [313.02, 12.01], ""bottom_right"": [553.98, 99.84],
+    ""label"": ""truck"", ""data_type"": ""ANNOTATION/BOUNDING_BOX""}]"
+    ```
+    The expected CSV format for this dataset should look like the CSV below:
+    ```
+    locator,image_id,ground_truths,extra_information_as_metadata
+    s3://my-images/intersection_1.png,1,"[{""top_left"": [538.03, 8.86], ... }, {""top_left"": ...}]",cloudy
+    s3://my-images/intersection_2.png,2,"[{""top_left"": [1380.35, 4.84], ... }]",rainy
+    ...
+    ```
+    At minimum, an object detection CSV should include a `locator` column as the ID field, and a `ground_truths`
+    column for the dataset's annotations. Additional metadata such as `image_id` or attributes about the weather
+    are useful but optional.
+
+    This CSV can be directly uploaded to Kolena's platform. To upload a dataset with the SDK, see details for
+    [structured data usage](../dataset/formatting-your-datasets.md/#structured-data).
+
+    See the [`object_detection_2d/upload_dataset.py`](https://github.com/kolenaIO/kolena/blob/trunk/examples/dataset/object_detection_2d/object_detection_2d/upload_dataset.py)
+    script in the code example for details on how a multiclass object detection dataset was generated and
+    an example for preparing a dataset file.
+
+=== "Web App"
+    To upload a dataset, having a properly formatted dataset file is a prerequisite.
+
+    To get started, navigate to [kolena.com](https://app.kolena.com/redirect/datasets) and
+    click `Import Dataset` then `Select From Cloud Storage`.
+    Using the explorer, navigate to `s3://kolena-public-examples/coco-2014-val/`
+    and select `coco-2014-val_person-detection.csv`. This is a dataset file for person detection using the COCO
+    dataset.
+
+    You will now see a preview of how the information is going to be consumed by Kolena.
+
+    Give your dataset a name and select `locator` as the ID field. The ID field uniquely identifies a datapoint
+    and is used when uploading model results to associate results with datapoints.
+
+    Click `Import` to create the dataset. Once the import has completed,
+    you can add descriptions and tags to organize your datasets.
+
+    <figure markdown>
+    ![COCO Dataset Upload](../assets/images/task-od-make-dataset-light.gif#only-light)
+    ![COCO Dataset Upload](../assets/images/task-od-make-dataset-dark.gif#only-dark)
+    <figcaption markdown>COCO Dataset Upload</figcaption>
+    </figure>
+
+=== "SDK"
+
+    The example code contains a script [`object_detection_2d/upload_dataset.py`](https://github.com/kolenaIO/kolena/blob/trunk/examples/dataset/object_detection_2d/object_detection_2d/upload_dataset.py)
+    which will process the CSV file `s3://kolena-public-examples/coco-2014-val/transportation/raw/coco-2014-val.csv`
+    and register a small transportation-based dataset in Kolena using the `register_dataset` function.
```

Comment:
`upload_dataset` — also should link [the API doc](https://docs.kolena.com/reference/dataset/?h=upload_dataset#kolena.dataset.dataset.upload_dataset).